### PR TITLE
Remove Windows binary jobs from blocking merge

### PR DESCRIPTION
**Pull request by huydhn**
It looks like there are more issues with Windows binary jobs after switching to non-ephemeral CPU runners than initially expected.  The jobs have been marked as unstable by https://github.com/pytorch/pytorch/pull/100581 and don't block merge on trunk anymore.

So let's also hide this on HUD like other unstable jobs while we're working on the fixes as it's quite confusing to look into HUD signals at the moment.  This can be reverted after https://github.com/pytorch/pytorch/pull/100581 is reverted.

Update: https://github.com/pytorch/pytorch/pull/100638 has been given the green light to not run Windows binary jobs on every PR as they are redundant and have high correlation with the rest of Windows CI jobs already run.

### Testing
https://torchci-git-fork-huydhn-hide-window-binary-build-fbopensource.vercel.app/

### CODE REVIEW

File: torchci/rockset/metrics/__sql/master_commit_red_avg.sql [hunk 1]
```diff
@@ -12,8 +12,7 @@ WITH all_jobs AS (
         AND job.name != 'generate-test-matrix'
         AND ( -- Limit it to workflows which block viable/strict upgrades
             ARRAY_CONTAINS(SPLIT(:workflowNames, ','), LOWER(workflow.name))
-            OR workflow.name like 'linux-binary%'
-            OR workflow.name like 'windows-binary%'
+            OR workflow.name like 'linux-binary%'
```
Comment (ZainRizvi) [inline torchci/rockset/metrics/__sql/master_commit_red_avg.sql:15]: updating this query will temporarily change the hud stats, but the stats will revert to including the "skipped" windows-binary failures once this change is reverted

File: torchci/rockset/metrics/__sql/master_commit_red_percent.sql [hunk 2]
```diff
@@ -14,7 +14,6 @@ WITH all_jobs AS (
         AND ( -- Limit it to workflows which block viable/strict upgrades
             ARRAY_CONTAINS(SPLIT(:workflowNames, ','), LOWER(workflow.name))
             OR workflow.name like 'linux-binary%'
-            OR workflow.name like 'windows-binary%'
```
Comment (ZainRizvi) [inline torchci/rockset/metrics/__sql/master_commit_red_percent.sql:17]: same

**Review by ZainRizvi — COMMENTED** [review]
Big Q: Do you want to also update the viable/strict upgrade logic to ignore failing windows-builds?

**Comment by huydhn** [comment]
> Big Q: Do you want to also update the viable/strict upgrade logic to ignore failing windows-builds?

I have the green light to not run windows binary build as part of the PR (redundant job, high correlation) in https://github.com/pytorch/pytorch/pull/100638, so I think the SQL change could become permanent here.  Hiding Windows binary job as unstable is not needed anymore, so I'll update the PR accordingly.

File: torchci/rockset/metrics/__sql/master_commit_red_avg.sql (hunk 1 above)
Comment (huydhn) [reply torchci/rockset/metrics/__sql/master_commit_red_avg.sql:15]: Update: https://github.com/pytorch/pytorch/pull/100638 has been given the green light to not run Windows binary jobs on every PR as they are redundant and have hight correlation with the rest of Windows CI jobs already run.

File: torchci/scripts/compute_correlation.py [hunk 3]
```diff
@@ -52,6 +52,13 @@ def compute():
         query_lambda="correlation_matrix",
         version=prod_versions["metrics"]["correlation_matrix"],
         workspace="metrics",
+        parameters=[
```
Comment (huydhn) [inline torchci/scripts/compute_correlation.py:55]: I computed correlation for `windows-binary-libtorch` workflow and accidentally added them here, but I guess it's ok to regenerate the correlation matrix anyway.

**Review by ZainRizvi — APPROVED** [review]

**Pull request merged**